psc-ide.el

```diff
@@ -371,33 +371,34 @@ use when the search used was with `string-match'."
                           ,@(psc-ide-server-command dir-name))))
 
 (defun psc-ide-server-command (dir-name)
-  "Tries to find the psc-ide-server-executable and builds up the
+  "Tries to find the purs executable and builds up the
   command by appending eventual options. Returns a list that can
   be expanded and passed to start-process"
   (let* ((npm-bin-path (if psc-ide-use-npm-bin
                            (psc-ide-npm-bin-server-executable)
                          nil))
-         (path (or npm-bin-path (executable-find psc-ide-server-executable)))
+         (path (or npm-bin-path (executable-find purs-executable)))
          (port (number-to-string psc-ide-port))
          (directory (expand-file-name dir-name))
-         (debug-flag (when psc-ide-debug "--debug"))
-         (globs (when (psc-ide--version-gte (psc-ide-server-version) "0.9.2") psc-ide-source-globs)))
+         (debug-flags (when psc-ide-debug '("--log-level" "debug")))
+         ;; (globs (when (psc-ide--version-gte (psc-ide-server-version) "0.9.2") psc-ide-source-globs))
```

Comment:
If we drop compatibility with all versions before `0.11.x` we don't need to perform this check anymore and can just pass the globs to the server command. I'm wondering if we still want to add an option to use the old command here, because most commercial users will still be on 0.10.x for a while.